### DISCUSSION TITLE
Not Compatible with Gnome 43

### DISCUSSION THREAD

**Discussion by craigccfl** [material-shell/material-shell#910] [Q&A] [answered]
Arch adopted Gnome 43 in a recent update. Material Shell extension appears to be incompatible with the new version of Gnome. Anyone find a work around besides not updating to the new version of Gnome? Fedora is also moving to Gnome 43. 

Know these things take some time, is a fix in the works?

**Comment by PapyElGringo** [comment]
The gnome 43 compatibility has been released few weeks ago so I don't know how did you installed it?

**Reply by craigccfl** [reply]
Cloned the install from this github site. I'll try a few things, see if I can get it running again. Really appreciate the work. Thanks for the update.  
![image](https://user-images.githubusercontent.com/38798834/199677706-338682bf-8fa9-4cc6-b6fe-a451da72e0bf.png)

**Reply by PapyElGringo** [reply]
Be sure to be on `main` branch then `pull` then `make install`

**Comment by craigccfl** [comment, marked as answer]
Removed and reinstalled, all good now. Will send some love your direction. Thanks for the information & keeping material-shell current.
